packages/fev4-fpm-writer/templates/action/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+	"sap.ui5": {
+		"extends": {
+			"extensions": {
+				"sap.ui.controllerExtensions": {
+					"sap.fe.templates.ObjectPage.ObjectPageController": {
+						"controllerName": "<%- controller %>"
```

Comment:
Technically this override now applies to all the ObjectPage, ideally we should strive to make this extension specific to one page

Reply:
Such as this sap.fe.templates.ObjectPage.ObjectPageController#sap.fe.core.fpmExplorer::Default

Reply:
Docu link : https://sapui5.hana.ondemand.com/#/topic/358cf2598d71462b8ac2bd8c944efbfd

Reply:
@nlunets like this: https://github.com/SAP/open-ux-tools/pull/39/commits/d9b50b4ddf1e008c24a5be1369bcca1ad638170a?